### PR TITLE
Updating rubocop things for June 2023

### PR DESCRIPTION
**Pull request by joseluistorres**
I'm proposing these changes to the current rubocop config because I highly recommend checking them from time to time to see if they still makes sense for the team.

Let me know if I can clarify any of these, thanks.

### CODE REVIEW

**Review by darwinwhaley — APPROVED** [review]
LGTM, although I'm not familiar with rubocop conventions yet.

File: config/default.yml [hunk 1]
```diff
@@ -7,11 +7,14 @@ Bundler/OrderedGems:
   Enabled: false
 
 Style/StringLiterals:
-  Enabled: false
+  Enabled: true
```
Comment (jmolineEA) [inline config/default.yml:10]: I'm fine with this in principle, but enabling it at this point (when projects have years of unenforced history) seems like it has the potential to create a lot of diff noise in PRs. I wouldn't say I feel strongly about it, but I'm not sure it's worth the hassle.

Comment (guillec) [reply config/default.yml:10]: I think @jmolineEA point is valid. 

I see three things we can do:
1. Leave it as `false` and ignore.
2. Leave it as `false` for now and next month do another update to just this parameter and see how much noise it does bring.
2. Commit to `true` and see if it does cause create a lot of noise with the agreement that we will revert it, if it is too annoying.

**Review by guillec — APPROVED** [review]
Left a comment on @jmolineEA comment but I am fine with any decision we take.

**Review by jmolineEA — APPROVED** [review]
Didn't mean for my comment to withhold approval.

**Review by farhajawed — APPROVED** [review]

File: config/default.yml (hunk 1 above)
Comment (joseluistorres) [reply config/default.yml:10]: @guillec @jmolineEA 
I agree with is definitely going to cause noise everywhere, but we need to start changing it little by little, I think this team has been really good to slowly do the changes and again, this just like any other rules are not written in stone and if we feel they don't make any sense later we can change them too.

**Pull request merged**